server/controllers/auth/signin.controller.js

```diff
@@ -35,53 +35,55 @@ const SigninController = {
     } = validateSignInInput(req.body);
 
     if (!isValid) {
-      return res.status(400).send({
-        status: 400,
-        error,
-      });
+      return res.status(400).send({ status: 400, error });
     }
 
+    // Convert email to lowercase
+    email = email.toLowerCase().trim();
```

Comment:
Remove this comment. The code is self-explanatory. Also, remove comments for any line of code that is self-explanatory.